src/app.js

```diff
@@ -8,6 +10,15 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.disable('x-powered-by');
 app.use(cookieParser());
+app.engine('handlebars', exphbs({ defaultLayout: 'main' }));
```

Comment:
remove this ? why you put it .. 
whats diffrent bettwen it and lines from 15 to 21

Reply:
done